Odata-docs/client/bulk-update-operations.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "Bulk Update Operations in OData Client"
+description: "How to perform bulk update operations in OData client"
+
+author: ElizabethOkerio
+ms.author: elokerio
+ms.date: 6/29/2023
+ms.topic: article
+ 
+---
+# Bulk Update Operations
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+OData Client [v7.17.0](https://www.nuget.org/packages/Microsoft.OData.Client/) introduced support for bulk update operations.
+
+## What is a bulk update operation 
+
+A bulk update operation is an operation that allows you to perform several operations (GET, POST and PATCH) on resources of the same type and all their related resources using a single request. This can be highly efficient in situations where you need to update multiple resources and wish to avoid sending separate requests to process each resource. [Learn More](https://devblogs.microsoft.com/odata/bulk-operations-support-in-odata-web-api/).
```

Comment:
Here's how "bulk update operation" is defined in the blog post that you referenced:

**A bulk operation is one that allows you to perform multiple operations (insert, update, delete) on multiple resources (of the same type) and their nested resources (to any level) with a single request.**

Are you able to perform a `GET` in a bulk update operation? How come here "delete" is not mentioned. Can you harmonize the doc and the blog post.

Reply:
It is suppossed to be post, patch and delete.